src/contracts/services/moneyMarket.ts

```diff
@@ -572,6 +572,83 @@ async function batchQueryMoneyMarketGetCollateral({
   }));
 }
 
+/**
+ * query the Public Logs for money market contracts
+ * NOT FOR PRODUCTION USE, CONTRACT IS IN DEVELOPMENT ON TESTNET ONLY
+ */
+/**
+ * Query the public events of a money market contract using RxJS
+ */
+function queryMoneyMarketPublicEvents$({
+  contractAddress,
+  codeHash,
+  lcdEndpoint,
+  chainId,
+  pagination,
+}: {
+  contractAddress: string,
+  codeHash?: string,
+  lcdEndpoint?: string,
+  chainId?: string,
+  pagination?: Pagination,
+}) {
+  const msgQueryMoneyMarketPublicEvents = (paginationParams?: Pagination) => ({
+    query_msg: {
+      public_events: {
+        pagination: paginationParams,
+      },
+    },
+  });
+
+  const parseMoneyMarketPublicEvents = (response: any) => ({
```

Comment:
move this parsing function to the upper scope, outside of the queryMoneyMarketPublicEvents$ function